### ISSUE TITLE
Invalid startup script for dnsmasq on macOS

### ISSUE THREAD

**Issue by dlzi** [TritonDataCenter/pkgsrc-legacy#461]
macOS 2016Q4, 64bits.

The dnsmasq package comes with a file (/opt/pkg/share/examples/rc.d/dnsmasq) that is not compatible with macOS. Is there a quick fix for that?

Wouldn't be more "native" for macOS packages to offer a launchctl(plist) example too?

**Comment by mamash** [comment]
Yes, the rc.d startup scripts that come with many packages are not intended for macOS at all - they work on BSD systems, and SMF manifests are often present for Illumos/SunOS systems.

Unfortunately there's no general support for launchd in pkgsrc, so until that is implemented, it's probably not worth investing time in providing a plist for dnsmasq. Sadly I'm not aware of anyone working on launchd support at this point.
